pipelines/resources/rclone.yaml

```diff
@@ -24,3 +24,52 @@ jobs:
   - put: concourse-rclone-resource-image
     params:
       build: concourse-rclone-resource
+      additional_tags: concourse-rclone-resource/tag
+  - task: tar-resource-files
+    config:
+      image_resource:
+        name: ''
+        source:
+          repository: node
+          tag: 14-buster-slim
+        type: docker-image
+      inputs:
+      - name: concourse-rclone-resource-image
+      - name: concourse-rclone-resource
+      outputs:
+      - name: concourse-rclone-resource-image
+      platform: linux
+      run:
+        # Inspired by https://github.com/concourse/ci/blob/master/tasks/build-resource-types-image.yml
+        args:
+        - -exc
+        - >-
+          echo "Packaging rclone";
+          cd concourse-rclone-resource-image;
+          tar czf rootfs.tgz --directory=rootfs .;
+          rm -rf ./rootfs/;
+          version=$(cat ../concourse-rclone-resource/tag);
+          echo { >> resource_metadata.json;
+          echo \ \ \"type\": \"rclone\", >> resource_metadata.json;
+          echo \ \ \"version\": \"${version}\", >> resource_metadata.json;
+          echo \ \ \"privileged\": false, >> resource_metadata.json;
+          echo \ \ \"version_history\": false >> resource_metadata.json;
+          echo } >> resource_metadata.json;
+          echo "resource_metadata.json contents:"
+          cat resource_metadata.json;
+          tar czf ./rclone.tgz rootfs.tgz resource_metadata.json;
+        path: sh
+  - task: packaged-resource-upload
```

Comment:
Might make sense to run this as a `put` step with the s3-sync resource so that the output can be tracked as a resource.